cronenbroguelike/commands.py

```diff
@@ -62,6 +64,31 @@ def look():
     _look()
 
 
+@when.when("random")
```

Comment:
`adventurelib` registers any `@when`-ed function that has been defined. So if you don't want this command to show up without the `random_run` flag, you can just protect the function definition behind an `if` statement.

Note line 105 below, where `extra_commands` is imported. You could do something like that: just put this in a separate source file and selectively import that file.

Reply:
Nice, thanks